README.md

```diff
@@ -1,12 +1,16 @@
-# STAC-geoparquet
+# stac-geoparquet
```

Comment:
If anything I'd have expected the opposite... for the colloquial name to be `stac-geoparquet` but for the official title to be `STAC-GeoParquet`